### PR TITLE
Add docker container to DH and GHCR

### PR DESCRIPTION
**Pull request by TheFrozenFire**
This PR adds a GitHub Action which automatically builds multi-platform docker images tagged for branches, PRs, semver tags, etc, pushing them to DockerHub and GitHub Container Repository.

Documentation for this workflow can be found here: https://github.com/docker/build-push-action

If merged, add a `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` secret to your GH repository with the corresponding credentials from DockerHub.

### CODE REVIEW

**Comment by shresthagrawal** [comment]
Maybe we should add a line in the README to use Kevlar using Docker?

**Comment by TheFrozenFire** [comment]
Added.

Also, if you want to test this before merging, I have the images building to [here](https://hub.docker.com/r/thefrozenfire/kevlar/tags).

File: README.md [hunk 1]
```diff
@@ -44,3 +44,10 @@ yarn start
 ```bash
 bash src/provers/light-optimistic/deploy-heroku.sh <heroku-app-name>
 ```
+
+### Deploy to Docker
+
+```bash
+docker run -p 8546:8546 --name kevlar shresthagrawal/kevlar
```
Comment (k4m4) [inline README.md:51]: @shresthagrawal Let's make sure to update this in case your `DOCKERHUB_USERNAME` isn't `shresthagrawal`.

**Review by k4m4 — APPROVED** [review]
Looks good to me. @shresthagrawal Let's add the `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` secrets and get this merged. Thanks for working on this, @TheFrozenFire!

File: README.md (hunk 1 above)
Comment (shresthagrawal) [reply README.md:51]: The username is correct

**Pull request merged**